### DISCUSSION TITLE
Interface scaling and Timeline position

### DISCUSSION THREAD

**Discussion by ota-works** [keiwando/toonsquid#506] [Ideas]
Hello, Keiwan!
Are there plans to enable interface scaling, depending on the user's preferences? I personally don't like having big buttons and prefer seeing most of my drawing space. I hope this is worth considering.

And lastly, are there also plans for allowing users to choose to have their timeline above the drawing canvas instead of the default position, which is below?
Sometimes, it feels like the timeline gets in the way of drawing, and I think giving users the chance to choose if they want the timeline above or below will really help those of us who love having the timeline above.

Thanks for taking the chance to read my suggestions.

**Comment by keiwando** [comment]
No immediate plans but I always take note of all of these suggestions and consider them for future updates.

**Reply by Lioncub901** [reply]
Really I really want that because I feel like there’s not much space to draw which Callipeg does have if that’s an advantage

**Comment by TempleLionStudios** [comment]
This is a great suggestion. I like adobe animates timeline. You often dont need to see what's on the frame. And moving frames around would be awesome. 

I do believe copying frames from several layers is coming in 2.0 right?

Our first episode of Temple Legends will be made entirely in ToonSquid 2.0

**Reply by Gottafindaname** [reply]
I hope it is!
